shell/shared/grain.js

```diff
@@ -303,11 +319,15 @@ if (Meteor.isClient) {
 
   Template.grainTitle.events({
     "click": function (event) {
-      var title = window.prompt("Set new title:", this.title);
-      if (title) {
-        var g = getActiveGrain(globalGrains.get());
-        if (g) {
-          g.setTitle(title);
+      var grain = getActiveGrain(globalGrains.get());
+      if (grain) {
+        var prompt = "Set new title:";
+        if (!grain.isOwner()) {
+          prompt = "Set a new personal title: (does not change the owner's title for this grain)";
```

Comment:
That might be a little verbose, but at least it sets expectations better.

I'm mildly worried how we'll express this if we move to making this an edit-inline field.  But I guess we can cross that bridge when we get there. :)